os/fs/smartfs/smartfs_utils.c

```diff
@@ -412,12 +412,12 @@ uint16_t get_used_byte_count(uint8_t *used)
 uint16_t get_leftover_used_byte_count(uint8_t *buffer, uint16_t base_index)
 {
 	uint16_t i, ending;
-	int sector_data_length = (CONFIG_MTD_SMART_SECTOR_SIZE - sizeof(struct smartfs_chain_header_s) - smart_sect_header_size);
+	int sector_data_length = (CONFIG_MTD_SMART_SECTOR_SIZE - CHAINHEADER_SIZE - smart_sect_header_size);
 	if (base_index >= sector_data_length) {
 		return sector_data_length;
 	}
 
-	base_index += sizeof(struct smartfs_chain_header_s);
+	base_index += CHAINHEADER_SIZE;
 	//We have to read from the last byte of the active chunk
```

Comment:
Can replace with /* */ comment style in accordance with coding guidelines

Reply:
I am sorry but it's not our coding guidelines.
```
Comment
[R01] Both comment styles are allowed : C89 "/* … */" style & C99 “//”style.
```

Reply:
This commit is only for Macro replacement.
In case there is a coding guideline violation, I can take it up later please.